CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing
+
+Thank you for considering making contributions to Cosmos-SDK and related repositories! Start by taking a look at this [coding repo](https://github.com/tendermint/coding) for overall information on repository workflow and standards.
+
+Please follow standard github best practices: fork the repo, branch from the tip of develop, make some commits, and submit a pull request to develop. See the [open issues](https://github.com/cosmos/cosmos-sdk/issues) for things we need help with!
```

Comment:
Can we link to issues tagged with "help-wanted" (and do a better job at tagging issues "help-wanted")?

Reply:
totally

Reply:
addressed